{"name": "test_check_fz", "status": "broken", "statusDetails": {"message": "AttributeError: module 'string' has no attribute 'captcha'", "trace": "def test_check_fz():\n        # cloud.send_device_datapoint('FZ', 10)\n        # time.sleep(60)\n>       device_data = cloud.get_device_detail()\n\ntest_send_dotapoint_check.py:26: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n..\\api\\IOT_CLOUD_API.py:91: in get_device_detail\n    login_result = self.login()\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <api.IOT_CLOUD_API.IOT_CLOUD_API object at 0x000001B75F1384F0>\n\n    def login(self):\n        ocr = ddddocr.DdddOcr()\n        captcha_result = self.get_captcha()\n        username = self.username\n        password = self.password\n        iot_url = self.iot_url\n        image_data = captcha_result['responseData']['image']\n        captcha_pic_base64_data = image_data.split(',')[1]\n        captcha = ocr.classification(None, captcha_pic_base64_data)\n>       captcha_new = string.captcha\nE       AttributeError: module 'string' has no attribute 'captcha'\n\n..\\api\\IOT_CLOUD_API.py:67: AttributeError"}, "start": 1656919409449, "stop": 1656919409765, "uuid": "fbc27d80-9d4f-40ef-bf93-745be7fbf708", "historyId": "31ce12057b51b2b2aa64411858c37bc5", "testCaseId": "9d060d588761426a3eb09cb001bbea6a", "fullName": "scripts.test_send_dotapoint_check#test_check_fz", "labels": [{"name": "parentSuite", "value": "scripts"}, {"name": "suite", "value": "test_send_dotapoint_check"}, {"name": "host", "value": "HONGCHA"}, {"name": "thread", "value": "12164-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "scripts.test_send_dotapoint_check"}]}